_rules/non-visual-reference-alternative-9bd38c.md

```diff
@@ -0,0 +1,433 @@
+---
+id: 9bd38c
+name: Content has non-visual reference alternative
+rule_type: atomic
+description: |
+  This rule checks that when content is identified through a visual reference, there are also non-visual references identifying the same content.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag20:1.3.3: # Sensory Characteristics (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G96: # Providing textual identification of items that otherwise rely only on sensory information to be understood.
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility Tree
+  - CSS Styling
+  - DOM Tree
+  - Language
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Daniël Strik
+    - Jean-Yves Moyen
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any [text node][] that is either [visible][] or [included in the accessibility tree][].
+
+## Expectation
+
+For each test target that includes at least one of the [visual reference words][], one of the following is true:
+
+- (**non-visual reference**) the test target is on the same [web page][] as a [textual][text] instruction that also identifies that [web content][] by a non-visual characteristic; or
```

Comment:
It's not clear to what "that web content" refers to (and this also applies to the other items in the list).

Ideally, the introductory sentence would specify what the visual reference words are used for (i.e. identify web content). But that is not possible because of the last 2 options (which do not identify content). Suggestion:

```suggestion
- (**non-visual reference**) the test target is on the same [web page][] as a [textual][text] instruction that identifies by a non-visual characteristic the same [web content][] identified by the test target; or
```

Reply:
Good catch. Fixing it a bit differently.
Moving "no indication" first, and frontloading "the test target gives indication" in each other item.